modules/jflip.cc

```diff
@@ -0,0 +1,291 @@
+/*
```

Comment:
This comment is no longer accurate - `jflip` isn't a skeleton any more. Please remove the first paragraph and the list and replace with a description of the module's function.

It's really important that documentation, including comments in source code, match the behavior of the program.